addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -821,6 +830,21 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         // We need to reload the existing type list.
         this.rerender = true;
     },
+    /**
+     * Generates new ids to avoid having several times the same id
+     * 
+     * @override
+     */
+    onClone() {
+        const originalInputEl = this.$target[0].querySelector('[id]:not(div)');
+        const cloneContainerEl = document.querySelectorAll(`#${CSS.escape(originalInputEl.id)}`)[1].closest('.s_website_form_field');
```

Comment:
Feels strange, you find the ID of the input in the clone, then you search for that ID and take the second you find... so you should refind yourself? :thinking: If am I wrong, definitely worth comments here.